docs/references/issue-rules/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Issue rules
+description: Improve the quality of your projects by detecting and fixing potential problems early with Nordcraft's issue rules.
+---
+
+# Issue rules
+
+Nordcraft's issue rules help you maintain high-quality standards in your projects by highlighting common problems.
+
+::: tip
+To learn more about where issue rules are shown in the editor, check out the [issues panel](/the-editor/issues-panel).
+:::
+
+::: tip
+To see how issue rules are implemented, visit the [Nordcraft GitHub repository](https://github.com/nordcraftengine/nordcraft/tree/main/packages/search/src/rules).
+:::
+
+## Actions
+
+### Project actions with no references
```

Comment:
Perhaps a table would be nicer to display this data?

Columns would be:

Name
Description
Supports auto-fixing